### ISSUE TITLE
Unable to ping connect.2b2t.org

### ISSUE THREAD

**Issue by clrxbl** [mcstatus-io/mcutil#6]
I currently use https://github.com/py-mine/mcstatus as my library of choice for this stuff but mcutil looked pretty interesting. However, both libraries seem to be unable to return data for connect.2b2t.org specifically, while something like https://mcsrvstat.us seems to be able to return data just fine.

```
panic: json: cannot unmarshal array into Go struct field .players.sample.id of type string

goroutine 1 [running]:
main.main()
        /var/folders/vk/g48d59z972gclyt6w1_z3szc0000gn/T/tmp.nnXYIAtM5u/main.go:19 +0xd4
```

**Comment by PassTheMayo** [comment]
This is something I'll definitely have to check into. If I get some time tonight, I will begin implementing a fix, otherwise it will have to be later this weekend. I will keep you updated.

**Comment by clrxbl** [comment]
Related: https://github.com/PaperMC/Velocity/pull/1165

**Comment by PassTheMayo** [comment]
This issue has been fixed in version `v3.1.1` of this library, and this fix is now live on `api.mcstatus.io` as well. Thank you for bringing this issue to my attention. I was also able to identify another bug with MOTD parsing when I was debugging this issue.
